kinematics_test.go

```diff
@@ -126,11 +127,27 @@ func BenchmarkInverseKinematics(b *testing.B) {
 		return 360 * rand.Float64()
 	}
 	for i := 0; i < b.N; i++ {
-		randomSeed := StepperTheta{randTheta(), randTheta(), randTheta(), randTheta(), randTheta(), randTheta()}
+		randomSeed := []float64{randTheta(), randTheta(), randTheta(), randTheta(), randTheta(), randTheta()}
 		desiredEndEffector := ForwardKinematics(randomSeed, AR3DhParameters)
 		_, err := InverseKinematics(desiredEndEffector, AR3DhParameters)
 		if err != nil {
-			b.Errorf("Failed inverse kinematics benchmark with: %s\nSeed: %f-%f-%f-%f-%f-%f", err, randomSeed.J1, randomSeed.J2, randomSeed.J3, randomSeed.J4, randomSeed.J5, randomSeed.J6)
+			b.Errorf("Failed inverse kinematics benchmark with: %s\nSeed: %f-%f-%f-%f-%f-%f", err, randomSeed[0], randomSeed[1], randomSeed[2], randomSeed[3], randomSeed[4], randomSeed[5])
```

Comment:
This seems less general than everything else, maybe it's fine?

Reply:
It's checking the errors of a 6 jointed robot, so when we have a 7 joint robot I'll check the 7. I'm pretty ok with test cases being specific.